asserts/model_test.go

```diff
@@ -1191,3 +1191,128 @@ func (mods *modelSuite) TestClassicWithSnapsMinimalDecodeOK(c *C) {
 		c.Check(noEssential.Size(), Equals, len(snaps)-1)
 	}
 }
+
+func (mods *modelSuite) TestValidationSetsDecodeFAIL(c *C) {
```

Comment:
we don't have tests called like this "FAIL", it also doesn't fit into golang naming conventions, they are usually called ..DecodeInvalid

Reply:
Ah right, I see that, changed it